website/search/elastic_search.py

```diff
@@ -26,6 +27,7 @@
 from website.search import exceptions
 from website.search.util import build_query
 from website.util import sanitize
+from website.search import file_util
```

Comment:
This should be between `from website.models import User, Node` and `from website.search import exceptions` - alphabetical order :)